src/core/pre.js

```diff
@@ -14,6 +14,10 @@ function getTypeTag(x) {
 }
 API.getTypeTag = getTypeTag;
 
+const hasOwn =
+  Object.hasOwn ||
```

Comment:
I think we can assume that `Object.hasOwn()` is present.

Reply:
Yeah we generally aim for ~2 years of backwards compatibility with mainstream browsers. If there were a bunch of iOS users stranded on an ancient version or something we might make special allowance for that. But I'm pretty sure this won't be the newest feature we're requiring.